examples/single_mem_node/main.rs

```diff
@@ -0,0 +1,199 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate raft;
+
+use std::sync::mpsc::{self, RecvTimeoutError};
+use std::time::{Duration, Instant};
+use std::thread;
+use std::collections::HashMap;
+
+use raft::*;
```

Comment:
Maybe we need to provide a prelude for those frequently used structs.

Reply:
Yes, we can add an issue with help-wanted.